CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### 🛠 Improvements
 - [7906](https://github.com/vegaprotocol/vega/issues/7906) - Connection tokens on the wallet survive reboot.
 - [8264](https://github.com/vegaprotocol/vega/issues/8264) - Add a command line on the wallet to locate the wallet files
+- [8026](https://github.com/vegaprotocol/vega/issues/8026) - Updare UPGRADING.md document
```

Comment:
```suggestion
- [8026](https://github.com/vegaprotocol/vega/issues/8026) - Update UPGRADING.md document
```

Reply:
That was already fixed with the first force push.